vms/platformvm/txs/executor/camino_tx_executor.go

```diff
@@ -1491,25 +1491,32 @@ func (e *CaminoStandardTxExecutor) BaseTx(tx *txs.BaseTx) error {
 		return err
 	}
 
-	if e.Bootstrapped.Get() {
-		baseFee, err := e.State.GetBaseFee()
-		if err != nil {
-			return err
-		}
+	caminoConfig, err := e.State.CaminoConfig()
```

Comment:
afaiu the following part should go to the top of the method
```go
	caminoConfig, err := e.State.CaminoConfig()
	if err != nil {
		return err
	}

	if !caminoConfig.LockModeBondDeposit {
		return e.StandardTxExecutor.BaseTx(tx)
	}
```

Reply:
Good catch, ty